HW_module_4_task2.py

```diff
@@ -0,0 +1,59 @@
+import string
+import random
+
+# create a list of random number of dicts (from 2 to 10)
+def create_list_of_dict(a, b, c=4, e=5, f=1):
+    dict_List = []
+    for x in range(random.randint(a, b)):
+        size = c
+        keys = random.sample(string.ascii_lowercase, size)
+        values = (random.randint(f, e) for y in range(size))
+        one_Dict = dict(zip(keys, values))
+
+        dict_List.append(one_Dict)
+    return dict_List
+
+dict_List = create_list_of_dict(a=2, b=10, e=50)               # call the function with parameters
+print('Initial list of dictionaries: ', dict_List)
+
+# get previously generated list of dicts and create one common dict
+# create the function with 1 parameter for dict_List
+def create_lists_with_keys(lst):
```

Comment:
Maybe it would be better if you do 2 functions here? Something like create_list_of_unique_keys and create_list_of_all_keys